### ISSUE TITLE
Will it be possible to run paperify in a MSYS64 environment?

### ISSUE THREAD

**Issue by SunsetMkt** [alisinabh/paperify#25]
I will try to create a Windows click-to-run package. This may be difficult.

**Comment by lwd-temp** [comment]
Yes, it's possible. But the package size would be terrible after extraction.

I'm not familiar with MSYS64 or MinGW, there may be a better solution.

**Comment by alisinabh** [comment]
Hello,

I don't know anything about this and don't have access to a windows machine. But It would be great if you can implement and I would be more than happy to review and merge it here. :)
